configure.scss

```diff
@@ -0,0 +1,53 @@
+$_n-ui-features: (
+	ads: false,
+	buttons: false,
+	grid: false,
+	colors: false,
+	icons: false,
+	typography: false,
+	cookieMessage: false,
```

Comment:
I guess we should distinguish between 'foundations' (starting to dislike that already) and the non-core, opt-in 'components'. Have tried to define them a little in [the wiki](https://github.com/Financial-Times/n-ui/wiki).